test/java/seedu/addressbook/util/PersonBuilder.java

```diff
@@ -0,0 +1,68 @@
+package seedu.addressbook.util;
+
+import seedu.addressbook.common.Utils;
+import seedu.addressbook.data.exception.IllegalValueException;
+import seedu.addressbook.data.person.Address;
+import seedu.addressbook.data.person.Email;
+import seedu.addressbook.data.person.Name;
+import seedu.addressbook.data.person.Person;
+import seedu.addressbook.data.person.Phone;
+import seedu.addressbook.data.tag.UniqueTagList;
+
+/**
+ * Factory class for building TestPerson
+ */
+public class PersonBuilder {
```

Comment:
I think we can do without a PersonBuilder. Level 2 a bit too early to do this kind of thing. Sorry I didn't think about it earlier.